docs/enterprise/concepts.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Concepts
+sidebarDepth: 1
+description: Learn how the Pomerium Enterprise Console works.
+---
+
+# Concepts
+
+## Namespaces
+
+In the Pomerium Enterprise Console, a namespace is where you can define an organizational unit of users and groups with fine-grained access management. This enables teams to self-service the routes and policies pertinent to them. Namespaces can optionally inherit from their parent units.
+
+## Service Accounts
+
+Service accounts handle machine-to-machine communication from Pomerium to your Identity Provider (**IdP**) in order to retrieve and establish group membership. Configuration is largely dependent on the IdP, but is usually an API acccess token with sufficient privlidges to read users and groups.
+
+## Routes
+
+Unlike the open-source Pomerium configuration, access is not defined alongside routing. Instead, authorization is configured by attaching [policies](#policies) to a route.
+
+## Policies
+
+In the open-source Pomerium config, routes and policies are configured in a single block, under `policy`:
```

Comment:
Should update this given recent changes from https://github.com/pomerium/pomerium/pull/2401.  We will be shifting this syntax under the `routes` block.

Reply:
IMHO, I should have been tagged on #2401 and updated the existing docs as part of the PR before it was merged.

It's not done until it's documented.